src/platform/silabs/rs911x/ble_config.h

```diff
@@ -0,0 +1,272 @@
+/*******************************************************************************
```

Comment:
This files should be removed all together. It litterally changes 7 defines that could easily be encapsulated inside an ifdef (in rsi_ble_config.h)

Reply:
this file has dependency in SDK , SDK is expecting a file with this name

../../../examples/lock-app/silabs/third_party/connectedhomeip/third_party/silabs/wifi_sdk/components/si91x/ble/src/rsi_ble_gatt_apis.c:18:10: fatal error: ble_config.h: No such file or directory
   18 | #include "ble_config.h"